src/components/input/input-default-styles.jsx

```diff
@@ -161,9 +161,11 @@ export default theme => {
         transition: transitions.create(['border-color', 'transform']),
         fontSize: 14,
         maxHeight: 27,
+        height: 21,
```

Comment:
Before we allowed inputs to resize depending on fontSize (Although up to a maximum of 27px for some reason). Lets check if this is still a valid use case!

Reply:
Maybe we can change this to `em` and remove `maxHeight`?

Reply:
Good idea. I will change to that.